woocommerce/woocommerce
# Product gallery/remove default

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/43476
Closes https://github.com/woocommerce/woocommerce/issues/43477

This PR does the following things:
- Removes the Product Gallery block from being the default on Single Product Template
- Adds the `Beta` label to the title of the block



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Appearance > Editor > Templates > Single Product and clear any customizations.
2. Edit the template and hover your mouse over the `Transform into blocks` action button and ensure in the preview, it is showing the legacy Product Image Gallery block.
3. Click on the transform button and ensure you see the legacy Product Image Gallery block.
4. Click on the Product Image Gallery block and `transforms to` action button and ensure you see the new Product Gallery (Beta) block on the list and if you hover over it, ensure you see the correct preview for the new block.
![file.png](https://github.com/woocommerce/woocommerce/assets/2132595/49a95b18-8026-47c4-9f3f-50aa01495bf9)
5. Click on it to transform. Ensure it transforms correctly. ( note I don't think we need a way to transform back as they can easily use the `undo` feature or undo customization ).
6. Remove all product gallery blocks new/old.
7. Click on the inserter and search for `product gallery` and ensure the new block is listed with the correct `(Beta)` label.
8. Add it to the template and save. Go to frontend and ensure it shows up correctly.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
